src/util.ts

```diff
@@ -89,9 +89,30 @@ export function throttleAfter<T extends (this: any, ...args: any[]) => void>(cal
 		let context = this;
 		let args: IArguments | null = arguments;
 
-		setTimeout(function () {
+		guaranteeMinimumTimeout(function () {
 			callback.apply(context, args);
 			args = context = ran = null;
 		}, delay);
 	};
 }
+
+export function guaranteeMinimumTimeout(callback: (...args: any[]) => void, delay?: number): Handle {
+	const startTime = Date.now();
+	let timerId: number | null;
+
+	function timeoutHandler() {
+		const delta = Date.now() - startTime;
+		if (delay == null || delta >= delay) {
+			callback();
+		} else {
+			timerId = setTimeout(timeoutHandler);
```

Comment:
Should this pass `delay - delta` as the second argument to `setTimeout()`?

Reply:
Yes, I goofed that when I changed it to make delay optional.  I'll fix it.